.ci_support/win_64_build_typedebugchannel_targetsconda-forge_mainopenssl1.1.1.yaml

```diff
@@ -1,5 +1,5 @@
 build_type:
-- release
+- debug
```

Comment:
This is pretty strange. This should be set to `release`. I wonder if there's a problem with conda-smithy?

Reply:
Yes, seems like some bug in smithy. But let's keep the discussion/work in one place for now, i.e., let's work on gh-597 first and backport those changes afterwards. Otherwise we're duplicating work (for you with copying changes, for reviewers looking in two places, for other maintainers checking their notifications).

Reply:
Sure, let's discuss on 597. I'll keep pushing any significant changes here in the meanwhile.